src/SWP/Bundle/FixturesBundle/Resources/themes/DefaultTheme/bundles/masonery.js

```diff
@@ -0,0 +1,502 @@
+/**
```

Comment:
this file seems to be not used and its location is wrong. It should be placed under `SWP/Bundle/FixturesBundle/Resources/themes/DefaultTheme/public/masonery.js`